manifests/init.pp

```diff
@@ -92,6 +95,17 @@
     validate_string($permitted_peer)
   }
 
+  if $umask {
+    validate_re($umask, '^0[0-7]{3}$',
+      "rsyslog::umask is <${umask}> and must be a valid four digit mode in octal notation with a leading zero.")
+  }
+
+  validate_re($file_create_mode, '^0[0-7]{3}$',
```

Comment:
The 

http://www.rsyslog.com/doc/rsconf1_umask.html
and
http://www.rsyslog.com/doc/rsconf1_filecreatemode.html

pages both says "The value given must always be a 4-digit octal number, with the initial digit being zero."